browser/main/lib/ConfigManager.js

```diff
@@ -58,11 +62,25 @@ function get () {
   let config = window.localStorage.getItem('config')
 
   try {
+    const homePath = global.process.env.HOME || global.process.env.USERPROFILE
+    const boostnotercPath = path.join(homePath, BOOSTNOTERC)
+    const boostnotercConfig = RcParser.parse(boostnotercPath)
+
     config = Object.assign({}, DEFAULT_CONFIG, JSON.parse(config))
+
+    if (boostnotercConfig !== undefined) {
+      config = Object.assign({}, DEFAULT_CONFIG, boostnotercConfig)
+      config.hotkey = Object.assign({}, DEFAULT_CONFIG.hotkey, boostnotercConfig.hotkey)
+      config.ui = Object.assign({}, DEFAULT_CONFIG.ui, boostnotercConfig.ui)
+      config.editor = Object.assign({}, DEFAULT_CONFIG.editor, boostnotercConfig.editor)
+      config.preview = Object.assign({}, DEFAULT_CONFIG.preview, boostnotercConfig.preview)
+    }
```

Comment:
If `RcParser.parse()` returns `{}`, you don't need to write this `if` statement.